### DISCUSSION TITLE
Add Whitelist feature to the domain blocking

### DISCUSSION THREAD

**Discussion by HotCakeX** [serverless-dns/serverless-dns#150] [Ideas]
I need to be able to use adblocking lists but also define some domains that will never get blocked even if they are included in some of the adblocker lists.

**Comment by ignoramous** [comment]
This feature is included in our client app for Android (https://github.com/celzero/rethink-app). For the resolver side, it isn't straight forward, but something I will implement in the coming months #21

**Reply by HotCakeX** [reply]
Thank you!
